components/Icon/Arrows/IconArrowDropDown.vue

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <IconBase
+        :width="size"
+        :height="size"
+        :icon-color="fillColor">
+        <polygon
+            :points="points"
+            :transform="iconTransform" />
+    </IconBase>
+</template>
+
+<script>
+import { Arrow } from '~/model/icons/Arrow';
+import IconBase from '~/components/Icon/IconBase';
+
+export default {
+    name: 'IconArrowDropDown',
+    components: {
+        IconBase,
+    },
+    props: {
+        state: {
+            type: String,
```

Comment:
`state` property give us only 4 rotate directions.
Maybe we can use the rotate property with values in degrees.
This way we can rotate the icon in all directions.